dev-libs/rocksdb/files/rocksdb-6.17.3-add_timer_for_riscv.patch

```diff
@@ -0,0 +1,38 @@
+Add timer for riscv.
+https://bugs.gentoo.org/834855
```

Comment:
Could you also mention upstream status and/or PR link in the patch header?

Reply:
basically, we'd hope patches could been pushed to upstream, so we carry less during next version bump (ease maintainer's burden, also be good open source citizen)

you can also put an upstream link or more verbose explanations in the patch header